docs/en/Repositories.md

```diff
@@ -87,6 +87,88 @@ If your entity is a soft-delete entity, you can use the `HardDeleteAsync` method
 
 See the [Data Filtering](Data-Filtering.md) documentation for more about soft-delete.
 
+## Bulk Operations
+You can execute bulk operations with `InsertMany`, `UpdateMany`, `DeleteMany` methods.
+
+Both providers `MongoDb` and `Ef Core` support bulk operations.
+
+### Customization
```

Comment:
This section if too advanced for this document. Instead, move this section to EF Core and MongoDB documents.
For EF Core, it can go under the Advanced Topics (https://docs.abp.io/en/abp/latest/Entity-Framework-Core#advanced-topics), as the last item. Similar for mongodb (https://docs.abp.io/en/abp/latest/MongoDB#advanced-topics)

Reply:
Can you check Commit 6be6470f